x-pack/plugins/cases/public/components/cases_context/cases_context_ui.tsx

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { getCreateCaseFlyoutLazyNoProvider } from '../../methods';
+import { CasesContextState } from './cases_context_reducer';
+
+export const CasesContextUI = React.memo(({ state }: { state: CasesContextState }) => {
```

Comment:
I think that the `CasesContextUI` is not that intuitive of what the purpose of this component is. I think something like `CasesGlobalComponents` or similar is more appropriate. The `context` usually refers to the React context.

Reply:
Good idea. I'll rename it.